source/image-handler/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# Define custom function directory
+ARG FUNCTION_DIR="/function"
+
+FROM node:16-bookworm
+
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+
+# Install build dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    g++ \
+    make \
+    cmake \
+    unzip \
+    build-essential \
+    ninja-build \
+    python3-pip \
+    meson \
+    bc \
+    wget \
+    libcurl4-openssl-dev \
+    libcgif-dev \
+    libheif-dev \
```

Comment:
libheif is the most important one, that is the one that deals with HEIC images. libvips will automatically include it if it is on the build system